### ISSUE TITLE
Name of recipe is working as a button but Voiceover doesn’t announce its role.

### ISSUE THREAD

**Issue by Karan4495** [dotnet/maui#5823]
GitHubTags:
#.NET MAUI; #A11ySev2; #A11yMAS; #A11yTCS; #Benchmark; #iOS; #iPad; #WCAG4.1.2; #GH_.NETMAUI_IOS_Mar2022; #Name Role Value​; #AINotInScope;

### Description

Name of recipe is working as a button but Voiceover doesn’t announce its role.

### Steps to Reproduce

1.Install and open ".NET MAUI" app.
2.Swipe till "search box".
3.Enter Recipe name in search box and search.
4.Verify Name of recipe is working as a button but Voiceover is announce its role or Not.

https://user-images.githubusercontent.com/100186395/161709164-31fb737f-7766-468f-b97c-05873d784cae.mp4

### Version with bug

Unknown/Other (please specify)

### Last version that worked well

Unknown/Other

### Affected platforms

iOS

### Affected platform versions

iOS 15.4(iPad)

### Relevant log output

```shell
Name of recipe is working as a button so Voiceover should announce its role i.e. when focus move to recipe name voiceover should announce recipe name and button.
```

**Comment by v-longmin** [comment]
@Karan4495 Could you share your repro project? Thanks.

**Comment by Karan4495** [comment]
Verified issue in ".NET MAUI" app. issue is fixed so closing the bug
IOS Version: iPad(15.5)
added attachment for references

https://user-images.githubusercontent.com/100186395/177997255-0501cc49-0dca-4aaa-9ba6-e73c705ecdbf.mp4
